src/test/mir-opt/inline/inline_shims.clone.Inline.diff

```diff
@@ -5,7 +5,7 @@
       debug f => _1;                       // in scope 0 at $DIR/inline-shims.rs:5:20: 5:21
       let mut _0: fn(A, B);                // return place in scope 0 at $DIR/inline-shims.rs:5:36: 5:44
       let mut _2: &fn(A, B);               // in scope 0 at $DIR/inline-shims.rs:6:5: 6:14
-+     scope 1 (inlined <fn(A, B) as Clone>::clone - shim(fn(A, B))) { // at $DIR/inline-shims.rs:6:5: 6:14
++     scope 1 (inlined <fn(A, B) as Clone>::clone - shim(<fn(A, B) as Clone>::clone)) { // at $DIR/inline-shims.rs:6:5: 6:14
```

Comment:
This change is fine, right?

Reply:
Yeah - as far as I know this is just a reflection of the `Display` impl change on `Instance` in compiler/rustc_middle/src/ty/instance.rs - I can revert the `Display` change if we're not happy with it.

Reply:
No, that's fine, just double checking. :)